### ISSUE TITLE
[Bug] Cannot chat with converted model of Qwen1.5 awq model. 

### ISSUE THREAD

**Issue by zhulinJulia24** [InternLM/lmdeploy#1492]
### Checklist

- [ ] 1. I have searched related issues but cannot get the expected help.
- [ ] 2. The bug has not been fixed in the latest version.

### Describe the bug

 Cannot chat with converted model of Qwen1.5 awq model. 

### Reproduction

1. lmdeploy lite auto_awq Qwen/Qwen1.5-7B-Chat --work-dir path-to/Qwen1.5-7B-Chat-inner-4bits

2. lmdeploy convert qwen path-to/Qwen1.5-7B-Chat-inner-4bits --model-format awq --group-size 128 --trust-remote-code

3. lmdeploy chat turbomind workspace --model-format awq

### Environment

```Shell
sys.platform: linux
Python: 3.10.14 (main, Mar 21 2024, 16:24:04) [GCC 11.2.0]
CUDA available: True
MUSA available: False
numpy_random_seed: 2147483648
GPU 0,1,2,3,4,5,6,7: NVIDIA A100-SXM4-80GB
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 11.7, V11.7.64
GCC: gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)
PyTorch: 2.1.0+cu118
PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201703
  - Intel(R) oneAPI Math Kernel Library Version 2022.2-Product Build 20220804 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v3.1.1 (Git Hash 64f6bcbcbab628e96f33a62c3e975f8535a7bde4)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX512
  - CUDA Runtime 11.8
  - NVCC architecture flags: -gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_90,code=sm_90
  - CuDNN 8.7
  - Magma 2.6.1
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.8, CUDNN_VERSION=8.7.0, CXX_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/c++, CXX_FLAGS= -D_GLIBCXX_USE_CXX11_ABI=0 -fabi-version=11 -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -DNDEBUG -DUSE_KINETO -DLIBKINETO_NOROCTRACER -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wall -Wextra -Werror=return-type -Werror=non-virtual-dtor -Werror=bool-operation -Wnarrowing -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=old-style-cast -Wno-invalid-partial-specialization -Wno-unused-private-field -Wno-aligned-allocation-unavailable -Wno-missing-braces -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_DISABLE_GPU_ASSERTS=ON, TORCH_VERSION=2.1.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=1, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

TorchVision: 0.16.0+cu118
LMDeploy: 0.4.0+
transformers: 4.40.0
gradio: Not Found
fastapi: 0.110.2
pydantic: 2.7.0
triton: 2.1.0
```

### Error traceback

```Shell
2024-04-25 09:23:20,518 - lmdeploy - WARNING - The sub command `lmdeploy chat turbomind` will be deprecated in future. Please use `lmdeploy chat` instead.
2024-04-25 09:23:20,518 - lmdeploy - WARNING - model_source: workspace
2024-04-25 09:23:20,518 - lmdeploy - WARNING - kwargs model_format is deprecated to initialize model, use TurbomindEngineConfig instead.
2024-04-25 09:23:20,518 - lmdeploy - WARNING - kwargs cache_max_entry_count is deprecated to initialize model, use TurbomindEngineConfig instead.
2024-04-25 09:23:20,518 - lmdeploy - WARNING - kwargs cache_block_seq_len is deprecated to initialize model, use TurbomindEngineConfig instead.
Traceback (most recent call last):
  File "/home/zhulin1/miniconda3/envs/v4/bin/lmdeploy", line 8, in <module>
    sys.exit(run())
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/lmdeploy/cli/entrypoint.py", line 37, in run
    args.run(args)
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/lmdeploy/cli/chat.py", line 131, in turbomind
    main(**kwargs)
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/lmdeploy/turbomind/chat.py", line 74, in main
    tm_model = tm.TurboMind.from_pretrained(
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/lmdeploy/turbomind/turbomind.py", line 396, in from_pretrained
    return cls(model_path=pretrained_model_name_or_path,
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/lmdeploy/turbomind/turbomind.py", line 162, in __init__
    self.tokenizer = Tokenizer(tokenizer_model_path)
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/lmdeploy/tokenizer.py", line 504, in __init__
    self.model = HuggingFaceTokenizer(model_folder)
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/lmdeploy/tokenizer.py", line 201, in __init__
    self.model = AutoTokenizer.from_pretrained(model_dir,
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/transformers/models/auto/tokenization_auto.py", line 862, in from_pretrained
    return tokenizer_class.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/transformers/tokenization_utils_base.py", line 2089, in from_pretrained
    return cls._from_pretrained(
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/transformers/tokenization_utils_base.py", line 2123, in _from_pretrained
    slow_tokenizer = (cls.slow_tokenizer_class)._from_pretrained(
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/transformers/tokenization_utils_base.py", line 2311, in _from_pretrained
    tokenizer = cls(*init_inputs, **init_kwargs)
  File "/home/zhulin1/miniconda3/envs/v4/lib/python3.10/site-packages/transformers/models/qwen2/tokenization_qwen2.py", line 179, in __init__
    with open(merges_file, encoding="utf-8") as merges_handle:
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Comment by zhulinJulia24** [comment]
I can chat with hf model. Check the difference of two config.ini, It seems max_batch_size value is not correct after convert.


![image](https://github.com/InternLM/lmdeploy/assets/145004780/8d62ddb6-b6e2-430b-ad52-f488ee271a14)

**Comment by zhulinJulia24** [comment]
I can chat with hf model successfully.

**Comment by AllentDan** [comment]
Fixed in v0.4.1
